src/utils.ts

```diff
@@ -1,59 +1,36 @@
-import { ResolutionType } from './typings';
+import { ImageListType } from './typings';
+
+export const triggerOpenDialog = (inputRef): void => {
+  if (inputRef.current) inputRef.current.click();
+};
+
+export const getAcceptString = (acceptType?: Array<string>) => {
```

Comment:
How about this function name?
```suggestion
export const getAcceptFileType = (acceptType?: Array<string>) => {
```